tests/services/test_service.py

```diff
@@ -100,7 +98,11 @@ def test_read_many_pid_values(app, search_clear, service, identity_simple, input
     Record.index.refresh()
 
     records = service.read_many(
-        identity_simple, ids=[item_one.id, item_two.id], fields=["metadata.type.type"]
+        identity_simple,
+        ids=[item_one.id, item_two.id],
+        # NOTE: "id" and "pid" are needed to produce the links in the result
+        #       Previous link generation would silently produce incorrect links
+        fields=["id", "pid", "metadata.type.type"],
```

Comment:
Discussion: yeah this is good and bad. Essentially the new way of generating links informs us of cases where urls were malformed... by throwing a BuildError. I don't think "id" and "pid" are universal though... If they are, we could add them to https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/records/service.py#L440 or make those default dumper fields configurable in `RecordServiceConfig`.